<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminState</class>
 <widget class="QWidget" name="AdminState">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="studentTable">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>401</width>
     <height>441</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>199</number>
   </attribute>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="studentLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Students: </string>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Student ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLine">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>110</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cLevelLabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Level:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cQuestLabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>250</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Quest:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cProgressLabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Time: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameLine">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLine">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>150</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="cLevelLine">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>210</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="cQuestLine">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>250</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="cProgressLine">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>290</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>450</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Student</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>450</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>View Stat Website</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logOutButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>490</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </widget>
  <widget class="QLabel" name="cPointsLabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>340</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Points:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cPointsLine">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>330</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
